EMSMobileSDK/EMSMobileSDK.swift

```diff
@@ -87,9 +87,10 @@ public class EMSMobileSDK: NSObject {
         self.applicationID = appID
         self.region = region
         if let userInfo = options?[UIApplication.LaunchOptionsKey.remoteNotification] as? [AnyHashable: Any] {
+            UserDefaults.standard.set("Init is called", forKey: "Initialize")
```

Comment:
Try to avoid using sentences for keys and values.
It would probably be best to use enums.